doc/source/conf.py

```diff
@@ -176,10 +176,14 @@ def _stop_fluent_container(gallery_conf, fname):
     "image_scrapers": ("pyvista", "matplotlib"),
     "thumbnail_size": (350, 350),
     "reset_modules_order": "after",
-    "reset_modules": (_stop_fluent_container),
+    "reset_modules": (_stop_fluent_container,),
```

Comment:
Can you fix the comment above?

Reply:
@prmukherj This comment `# Modules for which function level galleries are created. In`

Reply:
Done, thank you.